add FLIX logo (svg format)

harish551 opened the pull request.

JeremyParish69 commented:
Thank you. Could you please add the reference to the svg in the assetlists file?
https://github.com/OmniFlix/chain-registry/blob/71c71edfa486c0aafa0c8109a956b43bc679fc09/omniflixhub/assetlist.json#L22

harish551 commented:
> Thank you. Could you please add the reference to the svg in the assetlists file? https://github.com/OmniFlix/chain-registry/blob/71c71edfa486c0aafa0c8109a956b43bc679fc09/omniflixhub/assetlist.json#L22

hey @JeremyParish69 
Updated svg logo reference and added ibc connection details 
could you please check ?

harish551 commented on _IBC/omniflixhub-cosmoshub.json:
```diff
@@ -0,0 +1,31 @@
+      "connection_id": "connection-19"
+    },
+    "chain_2": {
+      "chain_name": "cosmoshub",
```
renamed file and updated required changes
